src/main/kotlin/no/nav/syfo/sykmelding/api/SyfoserviceApi.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.syfo.sykmelding.api
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.syfo.log
+import no.nav.syfo.papirsykmelding.tilsyfoservice.SendTilSyfoserviceService
+
+fun Route.registerSendToSyfoserviceApi(sendTilSyfoserviceService: SendTilSyfoserviceService) {
+    post("/api/sykmelding/{sykmeldingId}/syfoservice") {
+        val sykmeldingId = call.parameters["sykmeldingId"]!!
+        if (sykmeldingId.isNullOrEmpty()) {
+            call.respond(HttpStatusCode.BadRequest, "Sykmeldingid må være satt")
+        }
+
+        try {
+            sendTilSyfoserviceService.sendTilSyfoservice(sykmeldingId)
+        } catch (e: Exception) {
+            log.error("Kastet exception ved sending av sykmelding med id {} til syfoservice: {}", sykmeldingId, e)
+            call.respond(HttpStatusCode.InternalServerError, "Noe gikk galt ved sending til syfoservice")
+        }
+
+        call.respond(HttpStatusCode.OK)
```

Comment:
Hvordan respons er det man får her om det går galt? siden man har call.respond(HttpStatusCode.InternalServerError, "Noe gikk galt ved sending til syfoservice") noen linjer over?

Reply:
Jeg antok den kanskje bare ble kalt én gang... Men jeg er ikke sikker. Kan flytte den til try-blokken for sikkerhets skyld.